core/renderers/common/drawer.js

```diff
@@ -427,6 +453,24 @@ Blockly.blockRendering.Drawer.prototype.positionExternalValueConnection_ = funct
   }
 };
 
+/**
+ * Position the connection on an indented value input, taking into account
+ * RTL and the small gap between the parent block and child block which lets the
```

Comment:
Remove the comment on the small gap, since that doesn't apply for the base drawer.